orion/pkg/router/handler.go

```diff
@@ -22,7 +22,17 @@ func (h *Handler) SetupApiEndpoints() {
 		apiPrograms.POST("/v1/program/:programId", controllers.UpdateProgram)
 		apiPrograms.DELETE("/v1/program/:programId", controllers.DeleteProgram)
 	}
-	// apiClasses := router.Group("api/classes/")
+	apiClasses := h.Engine.Group("api/classes/")
+	{
+		apiClasses.GET("/v1/all", controllers.GetAllClasses)
+		apiClasses.POST("/v1/create", controllers.CreateClass)
+		apiClasses.GET("/v1/class/:classId", controllers.GetClassById)
+		apiClasses.GET("/v1/classes/program/:programId", controllers.GetClassesByProgram)
+		apiClasses.GET("/v1/classes/semester/:semesterId", controllers.GetClassesBySemester)
+		apiClasses.GET("/v1/classes/program/:programId/semester/:semesterId", controllers.GetClassesByProgramAndSemester)
+		apiClasses.POST("/v1/class/:classId", controllers.UpdateClass)
+		apiClasses.DELETE("/v1/class/:classId", controllers.DeleteClass)
```

Comment:
Move this endpoint and the UpdateClass endpoint to right below the GetClassById.
We want to keep similar api endpoint forms (`v1/class/:classId`) close to each other.
And then sort by Get -> Post -> Deletes.